api/v1alpha3/machineset_webhook_test.go

```diff
@@ -79,12 +79,12 @@ func TestMachineSetLabelSelectorMatchValidation(t *testing.T) {
 			}
 			if tt.expectErr {
 				err := ms.ValidateCreate()
-				g.Expect(err).To(gomega.HaveOccurred())
+				g.Expect(err).To(HaveOccurred())
```

Comment:
Since we aren't checking the actual value of err, other than it being not nil, we could test this similar to the way we are testing the else condition below:
```go
g.Expect(ms.ValidateCreate()).NotTo(Succeed())
g.Expect(ms.ValidateUpdate(nil)).NotTo(Succeed())
```

Reply:
Nice, fixing this in multiple places

Reply:
Done!